static/katex.less

```diff
@@ -553,4 +553,7 @@
         }
     }
 
+    .relpos {
+        position: relative;
+    }
```

Comment:
Just a though: Here the name describes *how* the group works, but not its *role*. Perhaps `positioning-root` or something to that effect would better characterize what this is about. On the other hand, it's overly verbose, so perhaps not.

One thing worries me (and has been worrying me for quite some time now): you introduce yet another class with specific meaning, and if the site into which we embed this class has other ideas of what that class name might signify, our element might receive some strange styling. This name in particular isn't too likely to get styled in a strange way, but I'd still be careful with every new class we introduce, as each such change might break some site.

Perhaps we should write a [less plugin](http://www.tutorialspoint.com/less/less_plugins.htm) to add some prefix to all our class names. That way we could use these same prefixes when building our trees, and simply tell page admins not to use any classes starting with that prefix for thir own purposes. `kt-` would be my prefix of choice, I think.